project.py

```diff
@@ -55,6 +69,16 @@ def __init__(self, n_slides=0,
         self.similarity = similarity
         self.__func_keyword_to_function__(compare_function)
         self.load_config()
+        self.section_threshold = section_thresh
+        self.reject_threshold = section_rjct
+        self.section_overlap = section_overlap
+        self.grid_size = grid_size
+        self.stored_section = [[[None for x in range(2)]
```

Comment:
@Foxiy well, `None for x in range(2)` is way longer (and more confusing) than `[None, None]`. I think you might reconsider changing that line.

I get that having three fors like this is kind of nice too, though.